### ISSUE TITLE
Paths broken on Windows

### ISSUE THREAD

**Issue by akrawitz** [teambition/gulp-ssh#70]
There was a breaking change in the code between v0.6.0 and v.0.6.1 when using gulp-ssh on Windows. Specifically, with commit 8982b3e, backslashes are no longer being converted to forward slashes.
`var outPath = path.join(destDir, file.relative.replace(/\\/g, '/'))`
should be something like:
`var outPath = path.join(destDir, file.relative.replace(/\\/g, '/')).replace(/\\/g, '/'))`
or, perhaps:
`var outPath = path.join(destDir, file.relative).replace(/\\/g, '/'))`
In other words, the replace needs to happen after joining the paths.

Thank you for this very helpful Gulp plugin.

**Comment by mojavelinux** [comment]
My mistake. I messed up the order of operations. I'll submit a follow-up PR to correct it. Stay tuned.

**Comment by mojavelinux** [comment]
It would be helpful if we could get the tests running on Travis and AppVeyor. That way, we can be sure we aren't breaking stuff.

We can use [nodegit](https://github.com/nodegit/nodegit) as a reference, since it tests SSH operations on both Travis and AppVeyor.
